spec/javascripts/integration/core/cluster_config_spec.js

```diff
@@ -38,16 +39,25 @@ module.exports = function(testConfigs) {
       }
 
       describe("with " + options.transport + ", forceTLS=" + options.forceTLS, function() {
+        var jasmineDefaultTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+
+        beforeAll(() => {
+          jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
```

Comment:
The ancient Jasmine didn't had any timeout, and now a couple of cases do timeout after the upgrade since there is default 5secs timeout. I honestly didn't dig into the test cases to see if it make sense to change test logic.